### ISSUE TITLE
Errors with directories/node modules

### ISSUE THREAD

**Issue by Kezmo** [FormidableLabs/react-native-ama#55]
Hi, i'm trying to get this building my project but i'm running into errors with different libraries this uses.
```
error: Error: Unable to resolve module wcag-color from /node_modules/react-native-ama/lib/internal/checks/contrastChecker.ts: wcag-color could not be found within the project or in these directories:
  node_modules/react-native-ama/node_modules
  node_modules
  ../node_modules

  1 | import React from 'react';
  2 | import type { StyleProp } from 'react-native';
> 3 | import { score } from 'wcag-color';
    |                        ^
  4 |
  5 | import { getPropertyFromStyle } from '../getPropertyFromStyle';
  6 | import { LogParams, getContrastCheckerMaxDepth } from '../logger';

```
The issue above is currently blocking me, ive added reanimated and gesture handler to my project as suggested in the docs but nothing related to this. Any ideas?

**Comment by Kezmo** [comment]
Updating this as i go to help anyone out in future. 
I've had to add multiple things to get by a few errors with missing libraries, imports and plugins that aren't in the docs.
`yarn add @opendevtools/wcag-color` to the project and then change an import inside the contrastChecker file to `import { score } from '@opendevtools/wcag-color';`
add the `'@babel/plugin-proposal-export-namespace-from'` plugin to the project
add the `'react-native-reanimated/plugin'` to the project

**Comment by retyui** [comment]
> Attempt to remove more RCTAppState listeners than added

This package doesn't use any `appStateDidChange` listeners , you have to the right library that cause the issue and update it (it should be fixed) if not create an issue in an appropriated repo

---

How debug?

1. open `node_modules/react-native/Libraries/EventEmitter/NativeEventEmitter.js`
2. find `removeListener` method
3. Add throw on JS side:

![image](https://user-images.githubusercontent.com/4661784/182787285-47e487ae-ec5c-4503-beb1-61d4b3da906e.png)


4. Then run app and you will see in stack trace of error the right library!

**Comment by ceceppa** [comment]
@Kezmo I'll try to include the `score` function inside the library itself, to avoid 3rd party dependency.
For the `appStateDidChange` as said by @retyui this event is not used by the library

**Comment by Coffeegerm** [comment]
I'm not seeing this issue for `wcag-color` but instead the `react-native-ama` package itself. 

When importing the `AMAProvider` component into my `App.tsx` file and importing `Text` into another file both raise the same `Cannot find module 'react-native-ama' or its corresponding type declarations.` error.

This doesn't appear to affect run time but it is failing tests.

**Comment by aglyadchenko** [comment]
Bumping this thread +1

**Comment by Coffeegerm** [comment]
Releases from last week seem to have cleared up the issue I mentioned above. Thanks to anyone involved with those releases!
